ui/components/app/assets/token-list/token-list.tsx

```diff
@@ -30,6 +31,7 @@ export default function TokenList({
   nativeToken,
 }: TokenListProps) {
   const t = useI18nContext();
+  const chainId = useSelector(getMultichainCurrentChainId);
   const { tokenSortConfig } = useSelector(getPreferences);
```

Comment:
Im seeing that to retrieve the chainId we sometimes use 
`const chainId = useSelector(getMultichainCurrentChainId);`
or
`const chainId = useSelector(getCurrentChainId);`

Is this something we can consolidate?

Reply:
Good shout @sahar-fehri ; I've updated to use `getMultichainCurrentChainId`